.github/workflows/interface-unit-tests.yml

```diff
@@ -42,6 +42,11 @@ on:
         required: false
         type: string
         default: "False"
+      pytest_store_durations:
```

Comment:
Set this argument to `true` in the `tests.yml` workflow to upload durations as artifacts.